tools/api/ontology.py

```diff
@@ -0,0 +1,58 @@
+import sys
+from typing import List
+
+sys.path.insert(1, '../validators/ontology_validator/')
+
+from yamlformat.validator.entity_type_lib import EntityType
+from yamlformat.validator.presubmit_validate_types_lib import ConfigUniverse
+from yamlformat import constants
+
+from model import StandardField
+from model import EntityTypeField
+
+class Ontology(object):
+    """Class defining an implementation of Google's ConfigUniverse as an ontology
+
+       Args:
+            universe: an instantiated ConfigUniverse Object
+
+       Attributes:
+            universe: A ConfigUniverse object detailing the various universes in the ontology
+
+       Returns:
+            An instance of Ontology class
+    """
+
+    def __init__(self, universe: ConfigUniverse):
+        super().__init__()
+        self.universe = universe
```

Comment:
Add TODO: Validate that universe has types expanded and fast-fail if not.

We can do this in a later PR